plenum/test/replica/test_api.py

```diff
@@ -93,20 +57,13 @@ def test_lst_sertificate_return_max_3PC_key_of_quorumed_prepare(replica):
     Therefore, expected Max3PC key must be (0, 1), because of previous prepare has enough quorum
     """
     replica.isMaster = True
-    replica.prepares.clear()
+    replica._ordering_service.prepares.clear()
     prepare1 = create_pre_prepare_no_bls(state_root='8J7o1k3mDX2jtBvgVfFbijdy6NKbfeJ7SfY3K1nHLzQB',
                                          view_no=0, pp_seq_no=1)
     prepare1.voters = ('Alpha:0', 'Beta:0', 'Gamma:0', 'Delta:0')
-    replica.prepares[(0, 1)] = prepare1
+    replica._ordering_service.prepares[(0, 1)] = prepare1
     prepare2 = create_pre_prepare_no_bls(state_root='EuDgqga9DNr4bjH57Rdq6BRtvCN1PV9UX5Mpnm9gbMAZ',
                                          view_no=0, pp_seq_no=2)
     prepare2.voters = ('Delta:0',)
-    replica.prepares[(0, 2)] = prepare2
-    assert replica.last_prepared_certificate_in_view() == (0, 1)
-
-
-def test_request_prepare_doesnt_crash_when_primary_is_not_connected(replica):
```

Comment:
Why remove this test?

Reply:
I think it was moved to plenum/test/consensus/order_service/test_orderer_api.py